src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -333,6 +336,16 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkDele
                 throw new NotFoundException();
             }
 
+            var ssoConfig = await _ssoConfigRepository.GetByOrganizationIdAsync(orgGuidId);
```

Comment:
This is for deleting users, not org @eliykat. Not sure what we're planning to do here?

Reply:
This has been removed, this is what I assumed at the time but I believe we're allowing it